### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by sstabley8385**

### CODE REVIEW

**Comment by sstabley8385** [comment]
Take it to the cell and hang it up

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# policy
+Tumblr policies and guidelines
+Get a look at this not what I would want to keep as a wife our wifey
```
Comment (sstabley8385) [inline README.md]: This is closed and should have been years ago

Comment (sstabley8385) [inline README.md]: No more back off are ill take you out

**Comment by 0509646999** [comment]
Love

**Comment by mibo420** [comment]
Respect one other!

**Comment by TimTheStuff** [comment]
Miss Kelly Josephine

**Comment by Asencio89** [comment]
From ef24996ec43f035fad96276fb14a9ed3e7ce7c6d Mon Sep 17 00:00:00 2001
From: sstabley8385 <sstabley8385@users.noreply.github.com>
Date: Wed, 9 Dec 2015 18:59:13 -0600
Subject: [PATCH] Create README.md

---
 README.md | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 README.md

diff --git a/README.md b/README.md
new file mode 100644
index 0000000..2389aaa
--- /dev/null
+++ b/README.md
@@ -0,0 +1,3 @@
+# policy
+Tumblr policies and guidelines
+Get a look at this not what I would want to keep as a wife our wifey

**Comment by KJandSB** [comment]
Get Outlook for Android

**Comment by pusuemg** [comment]
ty

File: README.md (hunk 1 above)
Comment (toppole69) [reply README.md]: Y

**Comment by QuangThong81** [comment]
1

**Comment by Armandd123** [comment]
K

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by macten30** [comment]
I dont understandt
Den 11/01/2019 17.47 skrev "apehdual87zzz" <notifications@github.com>:

**Pull request closed without merging**